docs/docs/use-static-query.md

```diff
@@ -45,7 +45,60 @@ export default () => {
 }
 ```
 
+### Advanced Example
+
+The potential of `useStaticQuery` is to make easier reuse queries across your code.
+
+One of the most used queries is to retrieve your site metadata.
+
+You can create a `useSiteMetadata` for that purpose:
```

Comment:
A classic example is to create a `useSiteMetadata` hook which will provide the `siteMetadata` to be re-used in _any_ component. It looks something like: